### PR TITLE
Update the translation in the file fr.po

### PR DESCRIPTION
**Pull request by Bafou**
Hello,
As we have said in our team account Lille1_opendev2015 on your forum we have update the translation that are on the fr.po file.

### CODE REVIEW

File: AndorsTrail/assets/translation/fr.po [hunk 1]
```diff
@@ -1026,7 +1026,7 @@ msgstr "Cela dépend de ce que vous attendez de moi."
 
 #: conversationlist_algangror.json:algangror_task2_4:3
 msgid "Something feels wrong here. I don't think I should help you."
-msgstr "Quelque chose ne m'inspire pas confiant, je ne crois pas que je devrais faire ça."
+msgstr "Quelque chose ne m'inspire pas confiantce je ne crois pas que je devrais faire ça."
```
Comment (DelvalletQuentin) [inline AndorsTrail/assets/translation/fr.po]: There's a small error here ;)

Comment (Bafou) [inline AndorsTrail/assets/translation/fr.po]: Good point changed that.

**Comment by Zukero** [comment]
While this looks very good overall, I'd rather have the changes done in launchpad. There are two reasons for that:
- Translations would keep being centralized, and upgradable by others.
- You wouldn't be able to mess with the "msgid" fields (occured on more than one occasion in this PR.
I agree working with launchpad is a pain. If you have some other online tool to propose, we'll be glad to try it out.

**Comment by Zukero** [comment]
While this looks very good overall, I'd rather have the changes done in launchpad. There are two reasons for that:
- Translations would keep being centralized, and upgradable by others.
- You wouldn't be able to mess with the "msgid" fields (occured on more than one occasion in this PR. 

I agree working with launchpad is a pain. If you have some other online tool to propose, we'll be glad to try it out.

**Pull request closed without merging**